ui/pages/notification-details/notification-details.tsx

```diff
@@ -116,3 +131,151 @@ export default function NotificationDetails() {
+                  style={{
+                    fontSize: '2.25rem',
+                    fontWeight: 'bold',
+                    color: '#60a5fa',
+                  }}
+                >
+                  {notification.data.length}
+                </p>
+                <p style={{ fontSize: '0.875rem', color: '#d1d5db' }}>
+                  total approvals
+                </p>
+              </Box>
+
+              <Box
+                style={{
+                  display: 'flex',
+                  flexDirection: 'column',
+                  gap: '0.25rem',
+                }}
+              >
+                <p
+                  style={{
+                    fontSize: '1.875rem',
+                    fontWeight: 'bold',
+                    color: '#fbbf24',
+                  }}
+                >
+                  {Math.min(5, notification.data.length)}
+                </p>
+                <p style={{ fontSize: '0.875rem', color: '#d1d5db' }}>
+                  are older than 14 days
+                </p>
+              </Box>
+
+              <Box
+                style={{
+                  display: 'flex',
+                  flexDirection: 'column',
+                  gap: '0.25rem',
+                }}
+              >
+                <p
+                  style={{
+                    fontSize: '1.875rem',
+                    fontWeight: 'bold',
+                    color: '#fb7185',
+                  }}
+                >
+                  {Math.min(2, notification.data.length)}
+                </p>
+                <p style={{ fontSize: '0.875rem', color: '#d1d5db' }}>
+                  have really large approval limits
+                </p>
+              </Box>
+              <p
+                style={{
+                  fontSize: '0.875rem',
+                  color: '#9ca3af',
+                  paddingTop: '0.5rem',
+                }}
+              >
+                Visit Portfolio to manage your tokens
+              </p>
+            </Box>
+          )}
+
+          <Box
+            width={BlockSize.Full}
+            display={Display.Flex}
+            flexDirection={FlexDirection.Row}
+            justifyContent={JustifyContent.spaceBetween}
+            padding={4}
+            gap={4}
+          >
+            <Button
+              variant={ButtonVariant.Primary}
+              externalLink={true}
+              size={ButtonSize.Lg}
+              width={BlockSize.Full}
+              endIconName={IconName.Arrow2UpRight}
+              href="https://revoke.cash/address/0xfBd4cdB413E45a52E2C8312f670e9cE67E794C37?chainId=1"
+            >
+              Visit Portfolio
+            </Button>
+          </Box>
+        </Box>
+      </Content>
+    </NotificationsPage>
+  );
+}
+
+export default function NotificationDetails() {
+  const { pathname } = useLocation();
+  const id = getExtractIdentifier(pathname);
+
+  if (id === TEST_REVOKE_NOTIFICATION_ID) {
+    return <RevokeNotificationDetails />;
```

Comment:
Ideally if this were a real notification, we can follow the `NotificationComponents` architecture to reuse a lot of logic.